dot/parachain/types/types.go

```diff
@@ -556,3 +560,27 @@ type ValidatorSignature Signature
 
 // Signature represents a cryptographic signature.
 type Signature [64]byte
+
+// BackedCandidate is a backed (or backable, depending on context) candidate.
+type BackedCandidate struct {
+	// The candidate referred to.
+	Candidate CommittedCandidateReceipt `scale:"1"`
+	// The validity votes themselves, expressed as signatures.
+	ValidityVotes []ValidityAttestation `scale:"2"`
+	// The indices of the validators within the group, expressed as a bitfield.
+	ValidatorIndices scale.BitVec `scale:"3"` // TODO: it's a bitvec in rust, figure out actual type
+}
+
+type ProspectiveParachainsMode struct {
```

Comment:
`ProspectiveParachainsMode` is also declared in collator protocol. You would want to remove the declaration from there and use this one everywhere.